src/views/RecIM/views/Admin/index.jsx

```diff
@@ -26,6 +39,12 @@ import { Typography } from '@mui/material';
 import recimLogo from 'views/RecIM/recim_logo.png';
 import { useNavigate } from 'react-router';
 import { deleteSport, getAllSports } from 'services/recim/sport';
+import SummarizeIcon from '@mui/icons-material/Summarize';
+import { LocalizationProvider, DateTimePicker } from '@mui/x-date-pickers';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { getRecIMReport } from 'services/recim/recim';
+import { Print } from '@mui/icons-material';
+//consider using react-to-print or react-pdf to create downloadable admin report
```

Comment:
```suggestion
//consider using react-to-print or react-pdf to create downloadable admin report
```

@bennettforkner do we have examples of react-pdf? I think you mentioned something about it at some point

Reply:
For context, I was considering using react-pdf, but ended up refactoring the styling and setup to be able to work with the default browser print dialog (by using divs, and css grids, instead of React elements).  I didn't end up using it, so I can delete this comment if you want me to.